lib/domain/dnd_controller.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:flutter_dnd/flutter_dnd.dart';
+
+part 'package:infinite_horizons/infrastructure/dnd_repository.dart';
+
+abstract class DndController {
+  static DndController? _instance;
+
+  static DndController get instance {
+    return _instance ??= _DndRepository();
+  }
```

Comment:
I think we can make this arrow function

Reply:
Could you please elaborate on this?

Reply:
` static DndController get instance => _instance ??= _DndRepository();`